ios/RNGoogleMobileAds/RNGoogleMobileAdsInterstitialModule.m

```diff
@@ -134,4 +133,15 @@ - (void)invalidate {
   }
 }
 
+RCT_EXPORT_METHOD(interstitialDestroy : (nonnull NSNumber *)requestId) {
+  [RNGoogleMobileAdsInterstitialModule removeInterstitial:requestId];
+}
+
+#pragma mark -
+
++ (void)removeInterstitial:(nonnull NSNumber *)requestId {
+  [interstitialMap removeObjectForKey:requestId];
+  [interstitialDelegateMap removeObjectForKey:requestId];
+}
+
```

Comment:
This is a) not this PRs fault and b) not something to take on in this PR but seeing the same code repeated over and over in both the java and the .m files makes me think "we have a lot of the same code for each ad type" which of course makes me think "we should probably have a base class that handles most of the "accounting" for keeping track of request ids and ads and destroying them etc, just with statics in the subclasses for the specific ad types and overrides / decorators when special behavior is needed

Reply:
Yes, in my previous library react-native-admob/admob, I removed duplicate codes via implementing common full screen ad class with generics, so we will have to refactor in the same manner for cleaner code.